modules/retail/pages/retail-formulas.adoc

```diff
@@ -230,14 +230,54 @@ The PXE formula is used to configure PXE booting on the branch server.
 . Navigate to the menu:Formulas[Pxe] tab, and set these parameters:
 * In the [guimenu]``Kernel filename`` field, keep the default value.
 * In the [guimenu]``Initrd filename`` field, keep the default value.
-* In the [guimenu]``Kernel commandline parameters`` field, keep the default value.
+* In the [guimenu]``Kernel command line parameters`` field, keep the default value.
+For more information about possible values, see <<retail.sect.formulas.pxe.kernelparams>>.
 * In the [guimenu]``PXE root directory`` field, enter the path to the saltboot directory (for example, [systemitem]``/srv/saltboot``).
 * In the [guimenu]``Branch id`` field, type a name to use as a branch identifier (for example: [systemitem]``Branch0001``).
 Use only alphanumeric characters for the branch identifier.
 . Click [btn]``Save Formula`` to save your configuration
 . Apply the highstate.
 
 
+
+[[retail.sect.formulas.pxe.kernelparams]]
+=== Kernel Command Line Parameters
+Saltboot supports common kernel parameters and saltboot-specific kernel parameters.
+All the parameters can be entered in the [guimenu]``Kernel Command Line Parameters`` field of the PXE formula.
+
+[systemitem]``kiwidebug=1``::
+Starts a shell on tty2 during boot and enables debug log level in Salt.
+
+[systemitem]``MASTER``::
+Overrides auto-detection of the Salt master. For example:
++
+----
+MASTER=myproxy.domain.com
+----
+
+[systemitem]``SALT_TIMEOUT``::
+Fall back to the local boot if the Salt master does not apply the saltboot state within this timeout (default: 60 seconds). For example:
++
+----
+SALT_TIMEOUT=300
+----
+
+// see https://bugzilla.suse.com/show_bug.cgi?id=1111348#c3
+[systemitem]``DISABLE_HOSTNAME_ID`` and [systemitem]``DISABLE_UNIQUE_SUFFIX``::
+Option for manipulating the names of the terminals.
```

Comment:
`DISABLE_UNIQUE_SUFFIX`

Setting this option to 1 disables adding random generated suffix to terminal minion id.

Please make sure your terminal has either unique hostname provided by dhcp and dns or terminal hardware comes with unique serial number stored in its SMBios memory.
Otherwise there is a risk of terminal minion id duplicity and failure to bootstrap the minion.

`DISABLE_HOSTNAME_ID`

If terminal has a hostname assigned by DHCP, it is by default used as a minion id. Setting this option to 1 disables this mechanism and SMBios information will be used as a minion id.

Reply:
ok